src/DIRAC/tests/Utilities/ProcessesCreator_withChildren.py

```diff
@@ -15,10 +15,10 @@ def do_sum(li):
 
 
 if __name__ == "__main__":
-    my_list = list(range(150000000))
+    my_list = list(range(15000000))
 
     pool = Pool(3)
-    pool.map(do_sum, [my_list[:50000000], my_list[50000000:100000000], my_list[100000000:]])
+    pool.map(do_sum, [my_list[:5000000], my_list[5000000:10000000], my_list[10000000:]])
```

Comment:
How about changing it to use iterators and avoiding the issue entirely?

```python
def do_sum(args):
    sum(range(*args))
    proc_name = current_process().name
    print(proc_name)
```

and

```suggestion
    pool.map(do_sum, [(0, 50000000), (50000000, 100000000), (100000000, 150000000)])
```